main.md

```diff
@@ -443,12 +599,192 @@ TBD
   </front>
 </reference>
 
+<reference anchor="ETSI" target="https://www.etsi.org/deliver/etsi_ts/119400_119499/119432/01.01.01_60/ts_119432v010101p.pdf">
+  <front>
+    <title>ETSI TS 119 432, Electronic Signatures and Infrastructures (ESI); Protocols for remote digital signature creation </title>
+     <author fullname="ETSI">
+	    <organization abbrev="ETSI">ETSI</organization>
+	  </author>
+   <date day="20" month="Mar" year="2019"/>
+  </front>
+</reference>
+
+<reference anchor="CSC" target="https://cloudsignatureconsortium.org/wp-content/uploads/2019/07/CSC_API_V1_1.0.4.0.pdf">
+  <front>
+    <title>Architectures and protocols for remote signature applications</title>
+    <author fullname="Cloud Signature Consortium">
+	    <organization abbrev="CSC">Cloud Signature Consortium</organization>
+	  </author>	
+   <date day="01" month="Jun" year="2019"/>
+  </front>
+</reference>
+
+<reference anchor="OpenID.CIBA"
+           target="https://openid.net/specs/openid-client-initiated-backchannel-authentication-core-1_0.html">
+        <front>
+          <title abbrev="CIBA">OpenID Connect Client Initiated Backchannel Authentication Flow - Core 1.0</title>
+          <author fullname="Gonzalo Fernandez Rodriguez" initials="G." surname="Fernandez">
+            <organization abbrev="Telefonica">Telefonica I+D</organization>
+          </author>
+          <author fullname="Florian Walter" initials="F." surname="Walter">
+            <organization abbrev="">Deutsche Telekom AG</organization>
+          </author>
+          <author fullname="Axel Nennker" initials="A." surname="Nennker">
+            <organization abbrev="">Deutsche Telekom AG</organization>
+          </author>
+          <author fullname="Dave Tonge" initials="D." surname="Tonge">
+            <organization abbrev="Moneyhub">Moneyhub</organization>
+          </author>
+          <author fullname="Brian Campbell" initials="B." surname="Campbell">
+            <organization abbrev="Ping Identity">Ping Identity</organization>
+          </author>
+          <date day="16" month="January" year="2019"/>
+        </front>
+      </reference>
+
 {backmatter}
 
+# Additional Examples
+
+## OpenID Connect {#openid}
+
+This examples tries to encapsulate all details specific to the OpenID Connect part of an authorization process into an authorization JSON object.
```

Comment:
I fear this will be taken as yet another way to make OIDC requests.

Reply:
that's what it is - the alternative is to stay with scope forever to turn on openid

Reply:
I see. If you think it's to sensitiv, we can also drop the example. FAPI could work an a OpenID extension to support authorization_details.

Reply:
I'd be happy with just the change (i.e., change "This examples" to "This hypothetical example") mentioned above.

Reply:
changed